src/components/browse/singleDatasetView/singleDatasetViewTabs/singleDatasetViewTabs.tsx

```diff
@@ -1,6 +1,6 @@
 import {
   faBookOpen,
-  faChartPie,
+  faBook,
```

Comment:
Please add a background for each of the 4 icons.

Reply:
Done

Reply:
Please add background for icons in the big boxes as well

Reply:
Done.